packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -176,6 +176,11 @@ export class VaMemorableDate {
 
     const [year, month, day] = (value || '').split('-');
 
+    // Add the `error-message` to the id list if there is an error
+    const describedBy = ["dateHint", error ? "error-message" : null]
+      .filter(id => !!id)
+      .join(" ");
```

Comment:
@TiffanyPender does this change anything about your ["dateHint" consideration](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/844#issuecomment-1179407375)?

Reply:
At the moment I'm trying to resolve the "Must" and "Should" categories of accessibility feedback. I do think that we could conditionally add the `error-message` id to the `aria-describedby`, at least for certain built-in errors, but if that's a "Consider" type of enhancement I'd rather focus on that later so that these components can be fully released sooner. In this case I think that would involve some state management which would take some time to develop and test well.

It sounds like I should remove these `aria-describedby` changes that impact `<va-memorable-date>` - is that right?

Reply:
Yes, as screen readers are not announcing the current `aria-describedby` text on each of the `<va-text-input>` field groups, I would remove the following changes:

> Since the memorable-date component already used the dateHint id for aria-describedby I wanted to have the attribute be:

> * dateHint when there wasn't an error, and
> * dateHint error-message when there was an error

It would be great to circle back on the `aria-describedby` specific to each field when time allows.